daprdocs/content/en/reference/components-reference/supported-state-stores/setup-dynamodb.md

```diff
@@ -70,6 +73,31 @@ In order to use DynamoDB TTL feature, you must enable TTL on your table and defi
 The attribute name must be defined in the `ttlAttributeName` field.
 See official [AWS docs](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html).
 
+## Partition Keys
+
+By default, the DynamoDB state store component will use the table attribute name `key` as primary/partition key in the DynamoDB table.
+This can be overridden by specifying a metadata field in the component configuration with a key of `partitionKey` and a value of the desired attribute name.
+
+To learn more about DynamoDB primary/partition keys, see the official [AWS DynamoDB Developer Guide.](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.CoreComponents.html#HowItWorks.CoreComponents.PrimaryKey)
+
+The following `statestore.yaml` file shows how to configure the DynamoDB state store component to use the partition key attribute name of `contractID`:
```

Comment:
Just so I am clear here on the scenario, what is the scenario for using the different partitionKey? Is this because I cannot rename the primary key in the database, or I am simply wanting to write a K/V entry with a different column? It would be good to put down at least 1 scenario where this is useful.